guides/create-graphql-mutation.md

```diff
@@ -0,0 +1,234 @@
+## At a glance
+
+You're working on an implementation for Leader of the Pack, an outdoor equipment retailer. In the GraphQL query guide we described data we added that contains whether we can ship a category of products to a particular zip. But now we need a way to update that information. For this we will create a GraphQL mutation
+
+## What you need 
+
+You need to understand what key you will use to query the data, and what the payload of the data is.
+
+## Identify which plugin owns the mutation
+
+When adding a new mutation, the first step is to decide which plugin should own it. This is usually obvious, but not always. You should think about whether any other plugins or services will need to call your mutation.
+
+## Define the mutation in the schema
+
+- If it doesn't already exist, create `schemas` folder in the plugin, and add an `index.js` file there.
+- If it doesn't already exist, create `schema.graphql` in `schemas` in the plugin.
+- Import the GraphQL file into `index.js` and default export it in an array:
+
+```js
+import importAsString from "@reactioncommerce/api-utils/importAsString.js";
+
+const schema = importAsString("./schema.graphql");
+
+export default [schema];
+```
+
+> NOTE: For large plugins, you can split to multiple `.graphql` files and export a multi-item array.
+
+- In the `.graphql` file, add your mutation within `extend type Mutation { }`. Add an `extend type Mutation` section near the top if the file doesn't have it yet.
+- Follow [Relay recommendations](https://relay.dev/docs/guided-tour/updating-data/graphql-mutations/) for mutation input arguments, which is to have only one argument named `input` that takes an input type that is the capitalized mutation name plus the suffix "Input", and to return a type that is the capitalized mutation name plus the suffix "Payload".
+
+   Example: `addAccountEmailRecord(input: AddAccountEmailRecordInput!): AddAccountEmailRecordPayload`
+
+- Add the Input and Payload types to the schema. Both must have `clientMutationId: String` field and may have any other fields as necessary. The mutation response payload should include whatever object was mutated.
+- Document your mutation, the new types, and all fields in those types using string literals.
+- If not already done, register your schemas in the plugin's `index.js` file:
+
+```js
+import schemas from "./schemas";
+
+export default async function register(app) {
+  await app.registerPlugin({
+    graphQL: {
+      schemas
+    },
+    // other props
+  });
+}
+```
+
+## Create the plugin mutation file
+
+- If it doesn't already exist, create `mutations` folder in the plugin, and add an `index.js` file there.
```

Comment:
Suggest dropping bullets from this lines as well. Seems to me that a step under a heading doesn't need bullets, whereas a step that has multiple sub-steps or things you need to consider within that step may benefit from bullets.